src/effects/VST3/VST3ParametersWindow.cpp

```diff
@@ -101,87 +265,86 @@ void VST3ParametersWindow::Fill(Steinberg::Vst::IEditController& editController,
                editController.plainParamToNormalized(parameterInfo.id, j),
                displayValue
             );
-            choice->AppendString(VST3Utils::ToWxString(displayValue));
+            list->AppendString(VST3Utils::ToWxString(displayValue));
          }
-         
-         choice->SetSelection(
-            static_cast<int>(editController.normalizedParamToPlain(parameterInfo.id, currentNormalizedValue))
-         );
-         choice->Bind(wxEVT_CHOICE, [this, id = parameterInfo.id](const wxCommandEvent& event) {
-            const auto comboBox = static_cast<wxComboBox*>(event.GetEventObject());
-            UpdateParameter(id, mEditController->plainParamToNormalized(id, comboBox->GetSelection()));
-         });
-         sizer->Add(choice, 0, wxEXPAND);
+         list->UpdateValue(editController);
+         list->Bind(wxEVT_CHOICE, &VST3ParametersWindow::OnParameterValueChanged, this);
```

Comment:
I see now that the mistake with combo boxes is fixed within the PR.  Still it would be better not to introduce it at all in the sequence of code transformations.